utils/types/QuickstartMutationVariable.ts

```diff
@@ -0,0 +1,81 @@
+export interface QuickstartAlertInput {
+  description?: string;
+  displayName: string;
+  rawConfiguration: string;
+  sourceUrl?: string;
+  type: string;
+}
+
+export interface QuickstartDocumentation {
+  displayName: string;
+  url: string;
+  description: string;
+}
+
+export interface DashboardScreenshot {
+  url: string;
+}
+
+export interface QuickstartDashboardInput {
+  description?: string;
+  displayName: string;
+  rawConfiguration: string;
+  sourceUrl?: string;
+  screenshots?: DashboardScreenshot[];
+}
+
+interface QuickstartAuthor {
+  name: string;
+}
+
+export interface QuickstartMetaData {
+  alertConditions?: QuickstartAlertInput[];
+  authors: QuickstartAuthor[];
+  categoryTerms?: string[];
+  description: string;
+  displayName: string;
+  slug?: string;
+  documentation?: QuickstartDocumentation[];
+  icon: string;
+  keywords?: string[];
+  sourceUrl?: string;
+  summary: string;
+  supportLevel?: string;
```

Comment:
These could probably be set in the type file:

```suggestion
  supportLevel?: 'COMMUNITY' | 'ENTERPRISE' | 'NEW_RELIC' | 'VERIFIED';
```